ddtrace/contrib/grpc/aio_client_interceptor.py

```diff
@@ -52,11 +53,15 @@ def _handle_add_callback(call, callback):
         callback(call)
 
 
-def _done_callback(span, code, details):
-    # type: (Span, grpc.StatusCode, str) -> Callable[[aio.Call], None]
+def _done_callback(span, code=None, details=None):
```

Comment:
do we need to even need to pass in code and details? being this is internal, we can change the function signature.

Reply:
yes, there are other wrappers that call this function.

Reply:
Ah, I think then it makes more sense here to leave the existing done callback as-is for the unary case, and add the necessary function for the broken streaming case.

Reply:
ok